src/core/core.tooltip.js

```diff
@@ -789,7 +789,10 @@ module.exports = function(Chart) {
 			// IE11/Edge does not like very small opacities, so snap to 0
 			var opacity = Math.abs(vm.opacity < 1e-3) ? 0 : vm.opacity;
 
-			if (this._options.enabled) {
+			// Boolean for empty tooltip
+			var hasToolTipContent = Boolean(vm.title.length || vm.beforeBody.length || vm.body.length || vm.afterBody.length || vm.footer.length);
```

Comment:
It's not required but I thought it made the code a little bit more readable. I can remove it if it doesn't